src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -24,6 +24,17 @@ public ClientAsyncAuthenticateTest()
             _log = TestLogging.GetInstance();
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "CI uses old Framework version, which doesn't support SystemDefault")]
```

Comment:
>"which doesn't support SystemDefault"

I would change this to something like "which doesn't support using SslProtocols.None for SystemDefaultTlsVersions behavior"

Reply:
Will change.